skyvern/webeye/actions/handler.py

```diff
@@ -1060,9 +1060,33 @@ async def fc_func(fc: FileChooser) -> None:
                 javascript_triggered=javascript_triggered,
             )
         ]
-        if await is_input_element(locator):
+
+        if skyvern_element.get_tag_name() == "label":
+            LOG.info(
+                "Chain click: it's a label element. going to try for-click",
+                task_id=task.task_id,
+                action=action,
+                locator=locator,
+            )
+            try:
+                if bound_element := await skyvern_element.find_label_for(
```

Comment:
what will happen if this isn't found?

Reply:
return None.

Reply:
interesitng -- in that case should we just keep doing what we used to do before?